index.html

```diff
@@ -142,7 +171,7 @@ <h2 class="project-title">Multi-Post Stories Gain+Glory</h2>
           <li class="language-tags">JavaScript</li>
           <li class="language-tags">html</li>
         </ul>
-        <button class="button">See Project</button>
+        <button class="button project-btn">See Project</button>
```

Comment:
- [ ] Hi @BahirHakimy, I would kindly suggest that all the project cards should be created dynamically and all the hardcoded written in the HTML file should be removed, With this, you'll be sure you followed the project requirement as much as possible. Kindly Fix it.

  - N.B: The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.** Check an example of a student's projects: [Link 1](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and [Link 2](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96).